README.md

```diff
@@ -57,7 +57,7 @@ If the image has a dark alternative, that's what you will get otherwise it will
 | links.reddit        | `String` | `false`    |         |                                                           |
 | links.youtube       | `String` | `false`    |         |                                                           |
 | links.instagram     | `String` | `false`    |         |                                                           |
-| donations           | `Object` | `false`    |         |                                                           |
+| donations           | `Object` | `false`    |         | { `"ethereum"`: "0x...", `"zilliqa"`: "zil1...", `"bsc"`: "0x..} |
```

Comment:
That's not how backticks work. Just copy paste what I sent you and put just two backticks wrapping the object.